### PR TITLE
Added argument to showUnlock method

### PR DESCRIPTION
**Pull request by endritbajrami**
The argument is a boolean value that indicates whether scan type should be NFC or QR.

### CODE REVIEW

**Comment by RafaRuiz** [comment]
Hello @endritbajrami , thanks for your contribution.
This is a breaking change for the javascript bridge function `showUnlock`, which didn't have any parameters earlier and will have them now.
The parameter `isNfc` is a boolean and before we assumed it was true, so can you please add the default parameter true to `isNfc` ?

File: index.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+import { NativeModules } from "react-native";
 
 const { RNDoordeckSdk } = NativeModules;
 
 const logout = () => RNDoordeckSdk.logout();
-const initDoordeck = (authToken, darkMode = true, closeButton = false) => RNDoordeckSdk.initDoordeck(authToken, darkMode, closeButton);
-const showUnlock = () => RNDoordeckSdk.showUnlock();
+const initDoordeck = (authToken, darkMode = true, closeButton = false) =>
+  RNDoordeckSdk.initDoordeck(authToken, darkMode, closeButton);
+const showUnlock = (isNfc) => RNDoordeckSdk.showUnlock(isNfc);
```
Comment (RafaRuiz) [inline index.js]: Breaking changes, please add this `isNfc` parameter as default (true)

**Comment by endritbajrami** [comment]
Sorry for the late response 😅

**Review by RafaRuiz — CHANGES_REQUESTED** [review]

**Comment by endritbajrami** [comment]
I also added this fix https://github.com/facebook/react-native/issues/35210 recommended by react-native team

**Review by RafaRuiz — APPROVED** [review]

**Pull request merged**